utils/renum/src/lepton-renum.c

```diff
@@ -394,7 +394,6 @@ void printhelp()
 void printver()
 	{
 	printf("This is lepton-renum, an advanced refdes renumber utility for Lepton EDA's lepton-schematic.\n");
-	printf("Version %s.  Lepton EDA version %s.%s\n",GRVERSION,
+	printf("Version %s. Lepton EDA version %s.%s\n",GRVERSION,
```

Comment:
I think the author added two spaces here not without reason.  Some people just used to it because it is recommended to do so in plain text manuals processed by roff or texinfo.  See for example the texinfo manual ([here](https://www.gnu.org/software/texinfo/manual/texinfo/html_node/Not-Ending-a-Sentence.html) and the next section).  Text processors like TeX/LaTeX just make a bit longer spaces between sentences.  And two spaces look more attractive in plain text.

Reply:
OK, reverted extra space.
Looks like a typo.